recipes/kissplice/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: kissplice
+  version: '2.4.0p1'
+
+source:
+  fn: kissplice-2.4.0-p1.tar.gz
+  url: "ftp://pbil.univ-lyon1.fr/pub/logiciel/kissplice/download/kissplice-2.4.0-p1.tar.gz"
+  md5: "36ef9e22a0607f0ab4516cc0d4cf9fea"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib {{CONDA_ZLIB}}*
+    - python 2.7.9
```

Comment:
please don't pin python here, skip it in the build section with `skip: True  # [py3k]`

Reply:
done